sql/2020/17_JAMstack/ssg_core_web_vitals_passing.sql

```diff
@@ -0,0 +1,85 @@
+#standardSQL
+# Core Web Vitals performance by CMS
```

Comment:
Is this the same as cdn_core_web_vitals_passing.sql? I see that it's querying 2020_07, but not sure if that's intentional.

Reply:
Should be 2020_08, thanks